docs/source/setup/installation.rst

```diff
@@ -31,11 +31,11 @@ To run the example notebooks locally, first clone the repository:
 
    git clone https://github.com/PyVRP/PyVRP.git
 
-Then, make sure your Python version has ``poetry``:
+Then, make sure your Python version has ``uv``:
```

Comment:
I think we can simplify this by just requiring the user to have `uv` installed:

```suggestion
Then, make sure to have [``uv``](https://docs.astral.sh/uv/getting-started/installation/) installed.
```
(and code block below)